### DISCUSSION TITLE
Flash still enable on regular search

### DISCUSSION THREAD

**Discussion by yriveiro** [folke/noice.nvim#635] [Q&A]
Hi,

I'm using LazyVim configuration with Noice and Flash, I notice that when I'm searching for stuff with the "/" command, Flash is still enabled and messing around with the search if I hit `s` or `r` keys.

Any way to disable Flash when searching on the traditional way?

**Comment by folke** [comment]
Please read the flash.nvim docs. This is the repo for noice. It's all in the docs.

**Comment by yriveiro** [comment]
@folke thank you for pointing me to the right place
